bindings/rust/standard/bench/src/s2n_tls.rs

```diff
@@ -325,11 +325,15 @@ impl TlsInfo for S2NConnection {
     }
 
     fn resumed_connection(&self) -> bool {
-        !self
-            .connection
-            .handshake_type()
-            .unwrap()
-            .contains("FULL_HANDSHAKE")
+        let handshake_type = self.connection.handshake_type().unwrap();
+        assert!(handshake_type.contains("NEGOTIATED"));
+        !handshake_type.contains("FULL_HANDSHAKE")
+    }
+
+    fn mutual_auth(&self) -> bool {
+        let handshake_type = self.connection.handshake_type().unwrap();
+        assert!(handshake_type.contains("NEGOTIATED"));
+        handshake_type.contains("CLIENT_AUTH")
```

Comment:
I think CLIENT_AUTH can appear when a client certificate message is sent containing no certificate? I think we'd still want to indicate no mutual auth in that case?

If this is the case, maybe checking `client_cert_used()` is better?